packages/actor-rdf-resolve-quad-pattern-federated/lib/FederatedQuadSource.ts

```diff
@@ -230,6 +231,7 @@ export class FederatedQuadSource implements ILazyQuadSource {
         let pattern: Algebra.Pattern | undefined;
 
         // Prepare the context for this specific source
+        this.contextDefault = getDataSourceContext(source, this.contextDefault);
```

Comment:
We don't want this to be overridden, since `contextDefault` is the context that is common across all sources.
Only in the scope of the current source do we want the source-specific context to be applied.

Reply:
On the next line, the actor also sets the 'source' field (also source-specific) to the same context. That's why I did it this way.
To what context should it be changed then?

Reply:
What you put in the context is correct, you just shouldn't override `this.contextDefault`. Instead, just put it in the local `context` directly.

Reply:
So the key-values need to be in this.contextDefault, but just not set in a way that requires an override?

Reply:
No, key-values should be in the local `context`.

Reply:
Nevermind, I got it